### PR TITLE
Add Lima Syndrome

### PR DESCRIPTION
**Pull request by PetalsOnWind**
Lima syndrome is a disorder that is opposite of Stockholm Syndrome

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -291,6 +292,25 @@ this disorder.
 
 [R: medicinet.com](https://www.medicinenet.com/munchausen_syndrome_by_proxy/article.htm)
 
+
+#### [Lima Syndrome](https://en.wikipedia.org/wiki/Stockholm_syndrome#Lima_syndrome)
```
Comment (icyflame) [inline readme.md]: Since this is closely related to Stockholm Syndrome, please add this entry immediately after that entry.

File: readme.md [hunk 2]
```diff
@@ -291,6 +292,25 @@ this disorder.
...
+
+> A form of Stockholm Syndrome where abductors develop sympathy for their hostage
+
+Lima Syndrome was named after the abduction at the Japanese embassy in Lima, Peru, 
+in 1996 when memebers of a militant group took hostage people attending a party 
+at the official residence of the Japanese ambassador, but set most of them free after 
+a few hours because of sympathy towards the hostages. Some of the reasons it could happen 
+is when one or more kidnappers don't agree with others,  don't feel like hurting 
+the hostages or the hostage is prone to developing sympathy. This disorder has been
+used in media mutiple times including Beauty and the Beast, A Song of Ice and Fire,
```
Comment (icyflame) [inline readme.md]: Nice pop culture references! :smile: :+1:

File: readme.md [hunk 3]
```diff
@@ -291,6 +292,25 @@ this disorder.
...
+is when one or more kidnappers don't agree with others,  don't feel like hurting 
+the hostages or the hostage is prone to developing sympathy. This disorder has been
+used in media mutiple times including Beauty and the Beast, A Song of Ice and Fire, 
+The World is Not Enough and Grand Theft Auto V.
+
+[R: Lima Syndrome](http://tvtropes.org/pmwiki/pmwiki.php/Main/LimaSyndrome)
+
+[R: A Tale of Kidnapping-From Stockholm to Lima Syndrome](http://www.ipsnews.net/1996/07/peru-tale-of-a-kidnapping-from-stockholm-to-lima-syndrome/)
+
+
```
Comment (icyflame) [inline readme.md]: You can put the scene in ASOIAF and Beauty and the Beast in which this syndrome was seen.

**Review by icyflame — CHANGES_REQUESTED** [review]

File: readme.md (hunk 3 above)
Comment (PetalsOnWind) [reply readme.md]: Made all the changes on Lima Syndrome

File: readme.md [hunk 4]
```diff
@@ -106,6 +107,27 @@ solve adaptive problems.
...
+at the official residence of the Japanese ambassador, but set most of them free after 
+a few hours because of sympathy towards the hostages. Some of the reasons it could happen 
+is when one or more kidnappers don't agree with others,  don't feel like hurting 
+the hostages or the hostage is prone to developing sympathy. This disorder has been
+used in media mutiple times including Beauty and the Beast where the Beast develops 
+sympathy and falls in love with his hosage, A Song of Ice and Fire where the Starks
+treated Theon, their hostage, as family, and Grand Theft Auto V where Trevor ends up 
+falling for Patricia Madrazo.
+
+[R: Lima Syndrome](http://tvtropes.org/pmwiki/pmwiki.php/Main/LimaSyndrome)
```
Comment (icyflame) [inline readme.md:126]: This is not a proper reference. I have spent some time searching for a worthwhile research work or a good reference detailing the events of that first kidnapping in Lima or something useful about the syndrome itself, but I have been unable to find anything. Please see if you can find something.

File: readme.md [hunk 5]
```diff
@@ -106,6 +107,27 @@ solve adaptive problems.
...
+
+[R: A Tale of Kidnapping-From Stockholm to Lima Syndrome](http://www.ipsnews.net/1996/07/peru-tale-of-a-kidnapping-from-stockholm-to-lima-syndrome/)
```
Comment (icyflame) [inline readme.md:128]: I am extremely confused about this reference. The story doesn't read like a news article and something seems off.

**Review by icyflame — CHANGES_REQUESTED** [review]
Thanks for re-arranging things.

I think we are better off replacing both references with something a little bit more academic, as any proper discourse about this syndrome is lacking. This syndrome only has an identity as Stockholm's reverse, I guess! :smile:

**Comment by icyflame** [comment]
Closing this for now. Please re-open after making the requested changes.

**Pull request closed without merging**